### PR TITLE
BW-1253: Auto-update helm chart upon CBAS release

### PR DESCRIPTION
**Pull request by fredlas**
Upon publishing a new version to this repo, github will now automatically write the tag associated with that new version into the [broadinstitute/cromwhelm repo](https://github.com/broadinstitute/cromwhelm)'s CBAS Helm Chart.

(Continued from https://github.com/DataBiosphere/cbas/pull/23 , because the presubmits don't work if you try to do a PR from a forked repo)

### CODE REVIEW

File: .github/workflows/publish.yml
Comment (cjllanwarne) [inline .github/workflows/publish.yml:113]: I'm not 100% sure on the difference and scoping between/across "jobs" and "steps"... did you confirm that this correctly references the `tag` step from the `publish-job` job?

File: .github/workflows/publish.yml
Comment (salonishah11) [inline .github/workflows/publish.yml:112]: This might need to be updated to look for version tag in file `cromwhelm/main/terra-batch-libchart/templates/_cbas-api.tpl` because of [this PR in cromwhelm](https://github.com/broadinstitute/cromwhelm/pull/48/files) that merges cromwell-helm with terra-batch-libchart

File: .github/workflows/publish.yml
Comment (salonishah11) [inline .github/workflows/publish.yml:115]: Same comment as above

File: .github/workflows/publish.yml
Comment (fredlas) [reply .github/workflows/publish.yml:112]: Thanks for the heads-up. I could make it accommodate both the old and new location, but probably makes sense to just block this on that one merging.

...but really this exposes the hackiness of the whole "bash commands to edit a version number in a text file" approach. It will probably break like this again eventually! I wonder if we can just leave off the version to have it default to "most recent". Not familiar with the syntax, whether that's a thing. I'll ask the team and investigate on my own if nobody knows.

Comment (fredlas) [reply .github/workflows/publish.yml]: Ok, a fruitful discussion with Janet indicates no, that is not a good approach. (It may or may not "work" mechanically speaking but is not a good idea and not how we do things).

**Review by aednichols — APPROVED** [review]

File: .github/workflows/publish.yml
Comment (fredlas) [reply .github/workflows/publish.yml:104]: Do you mean to write "cbas" rather than "composite-batch-analysis-service"? What is the different file you're referring to?

**Review by cjllanwarne — APPROVED** [review]
LGTM. Might be good to test-run with a test tag before merging finally

**Pull request merged**
